protocol/v2/qbft/controller/controller.go

```diff
@@ -12,11 +12,8 @@ import (
 
 	"github.com/bloxapp/ssv/protocol/v2/qbft"
 	"github.com/bloxapp/ssv/protocol/v2/qbft/instance"
-	ipfslog "github.com/ipfs/go-log"
 )
 
-var logger = ipfslog.Logger("ssv/protocol/qbft/controller").Desugar()
```

Comment:
I dont want to get rid of this logger until we do this https://bloxxx.atlassian.net/browse/IO1-2503
we will be using the ipfs log in order to achieve the path based configuration